tests/shell/shell_recorder/globalplugins.esr

```diff
@@ -1,32 +0,0 @@
-Mountpoint: system/
-DiffType: File
-
-
-< set system/elektra/globalplugins ""
```

Comment:
The tests now remove `system/elektra/globalplugins` before the test on my machine. Is the standard **error** output:

```
104: open	0000000001	di	0000000001
104: set	0000000026	di	0000000025	pos	presetstorage
104: set	0000000605	di	0000000579	pos	precommit
104: close	0000000677	di	0000000072
```

and

```
105: open	0000000000	di	0000000000
105: set	0000000021	di	0000000021	pos	presetstorage
105: set	0000000614	di	0000000593	pos	precommit
105: close	0000000662	di	0000000048
```

correct?